app/src/org/commcare/views/widgets/SpinnerWidget.java

```diff
@@ -124,18 +123,24 @@ public void setFocus(Context context) {
     // Defines how to display the select answers
     private class SpinnerAdapter extends ArrayAdapter<String> {
         final Context context;
-        String[] items = new String[]{};
+        String[] items;
         final int textUnit;
         final float textSize;
 
 
         public SpinnerAdapter(final Context context, final int textViewResourceId,
                               final String[] objects, int textUnit, float textSize) {
             super(context, textViewResourceId, objects);
-            this.items = objects;
             this.context = context;
             this.textUnit = textUnit;
             this.textSize = textSize;
+
+            //Creates an empty option to be displayed the first time the widget is shown
+            items = new String[objects.length+1];
+            items[0] = "";
+            for(int i = 1; i < items.length; i ++){
+                items[i] = objects[i-1];
+            }
```

Comment:
Linter says you should do `System.arraycopy(objects, 0, items, 1, items.length - 1);`. 
Using built-in copy methods is usually faster than doing a manual copying via loops.